proto/types/action.proto

```diff
@@ -308,9 +308,14 @@ message BlockEvmTransfer {
 // 2. Withdraw bucket from nsv2 protocol
 message ImplicitTransferLog {
   message Transaction {
-    string amount = 1;
-    string sender = 2;
-    string recipient = 3;
+    enum TxType {
+      EVM_TRANSFER = 0;
```

Comment:
maybe `EXECUTION` and `STAKE_WITHDRAW` reflect to their action name

Reply:
changed